terraform/module-repo-setup.tf

```
@@ -45,8 +46,8 @@ module "github_repository" {
   ))
   repository_name = each.value
   actions_secrets = {
-    ARM_CLIENT_ID: var.ARM_CLIENT_ID
-    ARM_SUBSCRIPTION_ID: var.ARM_SUBSCRIPTION_ID
-    ARM_TENANT_ID: var.ARM_TENANT_ID
+    ARM_CLIENT_ID : var.ARM_CLIENT_ID
+    ARM_SUBSCRIPTION_ID : var.ARM_SUBSCRIPTION_ID
+    ARM_TENANT_ID : var.ARM_TENANT_ID
```

Shouldn't the assignment have equal signs?

```suggestion
    ARM_CLIENT_ID       = var.ARM_CLIENT_ID
    ARM_SUBSCRIPTION_ID = var.ARM_SUBSCRIPTION_ID
    ARM_TENANT_ID       = var.ARM_TENANT_ID
```